package.json

```diff
@@ -9,6 +9,7 @@
     "generate:changelog": "lerna-changelog",
     "test": "npm-run-all build test:node test:browser lint",
     "test:node": "jasmine --config=jasmine.json && cd packages/linter && npm test",
+    "test:node:linter": "cd packages/linter && npm test",
```

Comment:
can you add it to the generic `test` command as well.

Reply:
I added a `test:linter` script, is that what you meant?